src/app/options/qgsoptions.cpp

```diff
@@ -486,10 +486,10 @@ QgsOptions::QgsOptions( QWidget *parent, Qt::WindowFlags fl, const QList<QgsOpti
   mCacheDirectory->setPlaceholderText( QStandardPaths::writableLocation( QStandardPaths::CacheLocation ) );
   mCacheSize->setMinimum( 0 );
   mCacheSize->setMaximum( std::numeric_limits<int>::max() );
-  mCacheSize->setSingleStep( 1024 );
-  qint64 cacheSize = mSettings->value( QStringLiteral( "cache/size" ), 256 * 1024 * 1024 ).toLongLong();
-  mCacheSize->setValue( static_cast<int>( cacheSize / 1024 ) );
-  mCacheSize->setClearValue( 50 * 1024 );
+  mCacheSize->setSingleStep( 50 );
+  qint64 cacheSize = mSettings->value( QStringLiteral( "cache/size2" ), 0 ).toLongLong();
```

Comment:
Maybe "cache/sizeInMiB" ?

Is there a way to put this string somewhere with a symbolic constant instead of repeating its value all over the place? (I realize that duplicating the string of the setting instead of using a symbolic constant is a habit I've noticed in the QGIS code base, so I'm OK if it is out of scope of this PR)